app/main/helpers/briefs.py

```diff
@@ -43,12 +43,26 @@ def domain(email):
 
 
 def is_supplier_eligible_for_brief(data_api_client, supplier_code, brief):
-    supplier = data_api_client. get_supplier(supplier_code)['supplier']
-    if brief.get('frameworkFramework') == 'dsp':
-        return len(supplier['services']) > 0
-    if brief.get('frameworkFramework') == 'dm':
-        return supplier.get('agreed_to_master_agreement', False)
-    return True
+    supplier = data_api_client.get_supplier(supplier_code)['supplier']
+    supplier_framework = supplier.get('frameworks')
+    new_seller = 'dm' in supplier_framework
```

Comment:
new_seller = False
 digital_marketplace_framework = data_api_client.req.frameworks('digital-marketplace').get()
        for framework in supplier.get('frameworks', []):
            if framework['framework_id'] == digital_marketplace_framework['frameworks']['id']:
                new_seller = True
        existing_seller = len(supplier.get('domains', {'legacy': []})['legacy']) > 0w